lucet-runtime/lucet-runtime-internals/src/alloc/tests.rs

```diff
@@ -2,9 +2,11 @@
 macro_rules! alloc_tests {
     ( $TestRegion:path ) => {
         use libc::c_void;
+        use rand::rngs::StdRng;
+        use rand_core::SeedableRng;
```

Comment:
```suggestion
        use rand::SeedableRng;
```
This is reëxported by `rand`, so you don't need the explicit `rand_core` dependency